docs/Changelog.md

```diff
@@ -43,7 +43,7 @@
 - Changes to the StopTimes call [#3576](https://github.com/opentripplanner/OpenTripPlanner/issues/3576)
 - Fix bug in optimize transfer service decorating path [#3587](https://github.com/opentripplanner/OpenTripPlanner/issues/3587)
 - Remove non-GBFS bicycle rental updaters [#3562](https://github.com/opentripplanner/OpenTripPlanner/issues/3562)
-
+- When importing Netex, allow bicycles on ferries by default [#3596](https://github.com/opentripplanner/OpenTripPlanner/pull/3596)
```

Comment:
As I commented on, for another PR the other day, we use 2 blank lines above and 1 below a heading for readability. You do not need to fix it, but keep it in mind for next time.